wallet/AndroidManifest.xml

```diff
@@ -96,7 +96,6 @@
             android:configChanges="keyboard|keyboardHidden"
             android:exported="true"
             android:label="@string/app_name_short"
-            android:launchMode="singleTask"
```

Comment:
Was there a problem with this solution?

Reply:
Yes, it resulted in multiple tasks or duplicate instances on the recent app lists when the app was started.

To solve the original issue, we also need to specify theses flags along with removing `singleTask`:
```
intent.addFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP or Intent.FLAG_ACTIVITY_CLEAR_TOP)
```

This is one of the side effects of using `taskAffinity=""` which prevents a particular vulnerability called Strandhogg v1.  Patching up this vulnerability created many problems for us.

Reply:
Gotcha.